### DISCUSSION TITLE
getting freeRTOS to work with pico w and ARduino IDE

### DISCUSSION THREAD

**Discussion by sw0rdd** [earlephilhower/arduino-pico#2000] [Q&A]
After I downloaded the board manager I get the pico to work with the arduino IDE but without freeRTOS 
when I use a simple freeRTOS program it compiles but I cant open the serial port and select the port dev/ttyACM0
I cant select port see this ![image](https://github.com/earlephilhower/arduino-pico/assets/144558784/d67ccd91-8bd2-4c49-9cca-5bcb0b7b358a)
here is my code in this 
![image](https://github.com/earlephilhower/arduino-pico/assets/144558784/115255a6-8f89-4abe-bc6d-c653cefa8b6c)
I am beginner to C/C++ and RTOS

**Comment by earlephilhower** [comment]
Try the latest release. 3.7.1  There was a FreeRTOS task switching corner case which was corrected.
